src/js/components/Collapsible.js

```diff
@@ -45,19 +45,21 @@ class Collapse extends Component {
       CLASS_ROOT,
       this.props.className
     );
-
     return <div {...this.props} className={classes} />;
   }
 };
 
 class Collapsible extends Component {
   render () {
     const Component = this.props.animate ? TransitionGroup : Box;
+    let collapseProps = Object.assign({}, this.props);
```

Comment:
Manny had a nice ES6 pattern for this where he enumerated all props and then used a ellipsis to capture the rest of them, as in `const { prop1, prop2, ...otherProps }`.